model/execution/DeliveryExecutionManagerService.php

```diff
@@ -346,6 +360,46 @@ public function isTimerAdjustmentAllowed($deliveryExecution)
         return true;
     }
 
+    /**
+     * Returns smaller time value for the current item/section/testPart/test chain
+     * @param string $deliveryExecutionId
+     * @return int
+     * @throws InvalidServiceManagerException
+     * @throws QtiTestExtractionFailedException
+     * @throws common_Exception
+     * @throws common_exception_Error
+     * @throws common_exception_NotFound
+     * @throws common_ext_ExtensionException
+     */
+    public function getAdjustedTime(string $deliveryExecutionId): int
```

Comment:
This method will return all timer adjustments, including time added as 'extendedTime' with `extended_time` LTI parameter. According to AC in ticket it must return time added by proctor separately.